components/Search/results/ResidentsTable.tsx

```diff
@@ -35,9 +35,13 @@ const ResultEntry = (person: LegacyResident): React.ReactElement => {
         {dateOfBirth && new Date(dateOfBirth).toLocaleDateString('en-GB')}
       </td>
       <td className="govuk-table__cell">
-        {address?.address && truncate(address.address, 4)}
+        {address?.address && truncate(address.address || '', 4)}
```

Comment:
Is `truncate` similar to a substring method and we want it to cut it down to four letters?

Reply:
Looks like it cuts down to a certain number of words, in this case 4. I haven't actually changed this, just made it work with a null `address.address` value